prime-dotnet-webapi/Services/interfaces/IReportingService.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading.Tasks;
+using System;
+
+namespace Prime.Services
+{
+    public interface IReportingService
```

Comment:
Can comments be added, possibly using text from PRIME-2475?

Reply:
comment added